packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html

```diff
@@ -16,9 +16,7 @@
         </div>
       </template>
       <template lwc:if={shouldDisplayPrintableUriLink}>
-        <a class="slds-truncate" href={hrefValue} aria-label={ariaLabelValue} target={target} onclick={handleClick} tabindex="0" role="link" data-test="link">
-          <c-quantic-result-text result={result} field="printableUri"></c-quantic-result-text>
-        </a>
+        <c-quantic-result-link engine-id={engineId} result={result} target={target} displayed-field="printableUri"></c-quantic-result-link>
```

Comment:
Didn't we have a case where instead of the clickUri the printableUri was supposed to display the "path" to the document for example in sub-folders?

Reply:
yeah we still have this case, it's just a bit higher in the code of this file
<img width="1248" alt="Screenshot 2024-02-26 at 2 39 41 PM" src="https://github.com/coveo/ui-kit/assets/86681870/d544baa6-fa6f-450b-a111-8bc766b97ed6">

Reply:
Oh gotcha it's purely in the html I missed it.